src/domains/fdr-common/env/weu-dev/terraform.tfvars

```diff
@@ -189,27 +189,18 @@ fdr_storage_account = {
 }
 
 fdr_re_storage_account = {
-  account_kind                       = "StorageV2"
-  account_tier                       = "Standard"
-  account_replication_type           = "LRS"
-  blob_versioning_enabled            = false
-  advanced_threat_protection         = false
-  advanced_threat_protection_enabled = false
-  public_network_access_enabled      = true
-  blob_delete_retention_days         = 30
-  enable_low_availability_alert      = false
-}
-
-fdr_history_storage_account = {
```

Comment:
isnt' use ?

Reply:
Not anymore, it was deleted after the massive infrastructure refactoring.